README.md

```diff
@@ -4,7 +4,68 @@ bookend is a classifier for text data. We are building a machine learning model
+> It was the best of times, it was the worst of times...
+
+will it know Charles Dickens wrote it?
+
+### Methodology
+
+We build many features out of the text, and use an ensemble machine learning model to predict the author.
+
+In the future we will expand to allow any-size snippets of text and additional authors, but for now we have focused on identifying 75-sentence chunks of text. We include the following authors, chosen as the top authors on Project Gutenberg who wrote their works in English (We exclude Shakespeare, as his works are largely plays rather than novels or short stories, and we exclude Poe, as his top works are largely collections that overlap with each other in content. We additionally add JK Rowling, who famously wrote the Harry Potter series as well as crime fiction novels under the pseudonym Robert Galbraith):
+
+| Author                | Jane Austen | Lewis Caroll | Daniel Defoe | Charles Dickens | Sir Arthur Conan Doyle | Jack London | J.K. Rowling | Mary Shelley | Robert Louis Stevenson | H.G. Wells | Oscar Wilde |
+|-----------------------|-------------|--------------|--------------|-----------------|------------------------|-------------|--------------|--------------|------------------------|------------|-------------|
+| Works in Training Set |             |              |              |                 |                        |             |              |              |                        |            |             |
+| Work(s) Testing Set   |             |              |              |                 |                        |             |              |              |                        |            |             |
+|                       |             |              |              |                 |                        |             |              |              |                        |            |             |
+
+Below we briefly summarize several avenues we investigated for engineering features from the text. For more detail, we refer you to the [notebooks](https://github.com/data-dart/bookend/tree/master/notebooks) and [source code](https://github.com/data-dart/bookend/tree/master/src).
+
+#### Bag-of-words
+
+We build a "vocabulary" of known words, assembled from the unique [lemmatized](https://en.wikipedia.org/wiki/Lemmatisation) words in the corpus. For each text sample, we build a feature vector indicating how many times each vocabulary word appeared.
+
+#### Lexical Features
+
+These features are a grab-bag of word and sentence level statistics, like the mean and variation in word and sentence length, and the [Flesch-Kincaid grade level](https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch%E2%80%93Kincaid_grade_level).
+
+#### Syntactic Features
+
+These features focus on the frequency of different parts of speech used by the authors.
```

Comment:
These features focus on the frequency of different [parts of speech] (https://en.wikipedia.org/wiki/Part_of_speech) used by the authors. We classify the words in a text sample into 7 distinct parts of speech categories and then build feature vectors from them. For example, for each text sample, we look at the frequency of nouns, a "noun-verb" bigram, and a "noun-verb-noun" trigram.

Reply:
Added.